charts/stardog/files/stardog.properties

```diff
@@ -1,2 +1,3 @@
 logging.audit.enabled=true
 logging.audit.type=text
+catalog.auto.create.db=false
```

Comment:
Why is the catalog db being disabled?

Reply:
Now that voicebox is enabled on all dbs (catalog included) it requires downloading some models. If this runs in an env without internet access it can fail at startup. The easiest workaround is to disable the catalog db auto creation. But instead of doing this in the default stardog.properties for any deploy like I did here, I now moved it and set it in the smoke tests themselves since we don't need the catalog db for those tests.